cookbook/bundles/override.rst

```diff
@@ -152,10 +152,17 @@ the constraints to a new validation group:
         <?xml version="1.0" encoding="UTF-8" ?>
         <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping
+                http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="FOS\UserBundle\Model\User">
+                <property name="plainPassword">
+                    <constraint name="NotBlank">
```

Comment:
why changing it? I think the length is a great example, as NotBlank is already part of the FOS User afaik